src/app/components/mno-confirm/mno-confirm.html

```diff
@@ -5,6 +5,10 @@ <h4 class="modal-title" translate translate-values="::modalOptions.headerTextExt
 <div class="modal-body">
   <p translate translate-values="::modalOptions.bodyTextExtraData">{{modalOptions.bodyText}}</p>
   <input ng-if="modalOptions.inputTextEnabled" ng-model="modalOptions.inputTextValue" type="text" class="form-control" placeholder="{{modalOptions.inputTextPlaceholder}}">
+  <div class="top-buffer-1" ng-if="::main.adminConfig.developerOnboardingURL()">
```

Comment:
I didn't see the file name on first review.
`mno-confirm` is a component used globally for all confirmation modal.

You sould make this modification more generic, eg:
```html
<div class="top-buffer-1" ng-if="modalOptions.additionalBodyHtml" ng-bind-html="modalOptions.additionalBodyHtml"></div>
```

Then when you call the modal you'd add:
```coffeescript
    if MnoeAdminConfig.developerOnboardingURL()
      modalOptions.additionalBodyHtml = \
        "<a href='#{MnoeAdminConfig.developerOnboardingURL()}' target='_blank'>" \
        + $translate.instant('mnoe_admin_panel.dashboard.product.create_product_modal.developer_onboarding_link_text') \
        + "</a>"

 MnoConfirm.showModal(modalOptions)
```

To simplify things you can make the whole text a link rather than splitting like on the mockups

Reply:
Makes sense, I'll do so, thanks.